Added Arabic and Swedish Languages

HEMO313 opened the pull request:
Made with love
By Moha and Nour

We tested everything

## Please put your Discord username so you can be contacted if a bug or regression is found:
Discord usernames:
hemo0
call_me_n0ur

Duwibi reviewed (changes requested):
I would suppose that these translations of the Arabic and Swedish languages are done well as I don't speak neither of the 2 languages. However, I don't approve the inclusion of the translations "Moha and Nour were here" and "Free Palestine" in both languages.

HEMO313 commented:
> I would suppose that these translations of the Arabic and Swedish languages are done well as I don't speak neither of the 2 languages. However, I don't approve the inclusion of the translations "Moha and Nour were here" and "Free Palestine" in both languages.

changed it yeah i speak both languages so i made sure they are all correct

Duwibi reviewed (changes requested):
I have reviewed the changes that were made and can confirm that the translations are now(almost) clear from unwanted translations. I see that you have assured me that all is translated well and everything has been checked but I still see the "Long live free Palestine" part left untouched in arabic, where it's written in the arabic script.

HEMO313 commented:
Ah must have missed it
it should be done now

michaelabilliot commented:
Im assuming from the past 2 attempts of you trying to put in palestine you put the lang_code and svg wrong too just to get more palestine in the file lmao
"svg": "ps" / probably a deliberate choice to put more palestine in but "sa" for saudi arabia is often more standard to use 
"lang_code": "ps" / ps is also wrong thats language code for Pashto not arabic (ar)
its overly condensed translation many descriptions in arabic are significantly shorter than their english counterparts omitting important detail

HEMO313 commented:
> Im assuming from the past 2 attempts of you trying to put in palestine you put the lang_code and svg wrong too just to get more palestine in the file lmao "svg": "ps" / probably a deliberate choice to put more palestine in but "sa" for saudi arabia is often more standard to use "lang_code": "ps" / ps is also wrong thats language code for Pashto not arabic (ar) its overly condensed translation many descriptions in arabic are significantly shorter than their english counterparts omitting important detail

when was the second attempt? thats first, second whats wrong with putting free Palestine are you mad about that? second whats wrong with having the palestine flag for the Arabic langauge every Arab will agree with that whats the thing with Palestine making you so mad?? third lang_code: ps or whatever i put here it wont change a shit in the game how about "ar-PS"

last thing ofc it will be shorter in Arabic cause Arabic is not same as English have you ever heard that before? do you speak even Arabic or are u using translator have you ever seen other language like germany have too long words and the same word in English have shorter word

michaelabilliot commented:
please be calm im addressing issues that i see with the file, i had my native arabic speaking friend check the files and im relaying his comments cause he doesnt have a github account

my issue isnt putting free palestine you can gladly do that and ill support it but translating it wrong just to put "free palestine" is my issue translations should be accurate without random mistranslations in there 

i already talked to nour about this please stop having outbursts here and on the translation discord

HEMO313 commented:
> please be calm im addressing issues that i see with the file, i had my native arabic speaking friend check the files and im relaying his comments cause he doesnt have a github account
> 
> my issue isnt putting free palestine you can gladly do that and ill support it but translating it wrong just to put "free palestine" is my issue translations should be accurate without random mistranslations in there
> 
> addressing the lang_code: part ps is a literal code for a different language its not "palestine" its pashto its inaccurate to put ps or ar-ps because youre not translating the file to arabic-pashto youre translating it to just arabic (ar)
> 
> i already talked to nour about this please stop having outbursts here and on the translation discord

nah i'm not having any outbursts but the way you say it shows clearly that you have a issue with the word Palestine or smth we made sure the translation 100% right me and nour you are now saying you are translating it wrong just to put Free Palestine make sure the (Arab) friend you asked knows really Arabic cause the translated Arabic in the file is native Arabic the original one not a dialect, i started feeling you started hating for seeing free palestine no worries bro you will see it often

michaelabilliot commented:
im a muslim from turkey i dont have an issue with palestine i have an issue with you using it wrongly and having mistranslations?

HEMO313 commented:
> im a muslim from turkey i dont have an issue with palestine i support free palestine
> 
> i have an issue with you using it wrongly and having mistranslations?

tbh i don't really care where you are from + it wasnt my idea to add free palestine thing and if u ask any Arab person who will use the language in openfront he will have no problem having it and the translation is not wrong as i said before and you are turk isn't that stupid for you to say that this translation is wrong and you don't even speak Arabic you asked a friend or you used chatgpt to help you? and its not your choice to say this translation is wrong or not its for the dev to say

michaelabilliot commented:
i already talked to nour and sorted everything out so please stop arguing further thank you :D

HEMO313 commented:
> i already talked to nour and sorted everything out so please stop arguing further thank you :D

nah who said i'm arguing i'm responding to the things u are saying
